Add docs for service accounts 8.5 (#46801)
grafana/grafana: docs/sources/administration/service-accounts/enable-service-accounts.md

---
title: 'Enable service accounts in Grafana'
menuTitle: 'Enable service accounts'

> description: 'This contains information to enable service accounts feature in Grafana'

Reviewer: ```suggestion
description: 'This topic shows you how to to enable the service accounts feature in Grafana'
```

aliases: [docs/sources/administration/service-accounts/enable-service-accounts.md]
weight: 40
keywords:
  - Feature toggle
  - Service accounts
---

> ## Enable service accounts in Grafana

Reviewer: ```suggestion
# Enable service accounts in Grafana
```

Service accounts are available behind the `service-accounts` feature toggle in Grafana 9.0+.

> You can enable it either in a [config file]({{< relref "../../administration/configuration.md#config-file-locations" >}}) or by [configuring an environment variable]({{< relref "../../administration/configuration/#configure-with-environment-variables" >}}).

Reviewer: ```suggestion
You can enable service accounts by:
- modifying the Grafana configuration file, or
- configuring an environment variable
```

> ### Enable service accounts with configuration file

Reviewer: ```suggestion
## Enable service accounts with configuration file
```

> In your [config file]({{< relref "../../administration/configuration.md#config-file-locations" >}}), add `service-accounts` as a [feature_toggle]({{< relref "../../administration/configuration.md#feature_toggle" >}}).

Reviewer: I would approach this task as you've done with others: provide an intro sentence, a before you begin section, a stem sentence, and then steps.  For intro sentence, I would go with a very basic "This topic shows you how to enable service accounts by modifying the Grafana configuration file."

The Before you begin section, what to include in there? Does the user need to power down the Grafana server? Do they need special permissions to access the Grafana server?

The steps can be something like:

1. Sign in to the Grafana server and locate the configuration file.
     For more information about finding the configuration file, refer to LINK.

2. Open the configuration file and locate the [feature toggles] section.
3. In the [feature toggles] section, add the following parameter:

```
[feature_toggle]
# enable features, separated by spaces
etc.
etc.
```

4. Save your changes and restart the Grafana server. (Making this up, not sure if a restart is required).

```
[feature_toggles]
# enable features, separated by spaces
enable = service-accounts
```

> ### Enable service accounts with an environment variable

Reviewer: ```suggestion
## Enable service accounts with an environment variable
```

> You can use `GF_FEATURE_TOGGLES_ENABLE = service-accounts` environment variable to override the configuration file.

Reviewer: Use the same approach that I described above to add an intro, before you begin, stem sentence, and steps.
